evap/static/font-awesome/scss/_mixins.scss

```diff
@@ -70,4 +70,3 @@
     content: unquote("\"#{ $fa-var }\"");
   }
 }
-
```

Comment:
In one of the earlier files, you added a final newline. This now removes it (I think?)